apps/core/static/cosmos/css/about.css

```diff
@@ -44,4 +69,9 @@
 
 .partners .carousel-item {
     justify-content: center;
+}
+
+.testimonial-text {
+    font-size: 14px;
+    font-style: italic;
```

Comment:
I think this is possible to do with the bootstrap classes as well, which is nicer. We try to use as little manual css as possible, and use the bootstrap stuff for everything.